src/Quasar/Error.purs

```diff
@@ -49,6 +52,7 @@ printQError = case _ of
   Forbidden → "Resource is unavailable, the current authorization credentials do not grant access to the resource"
   PaymentRequired → "Resource is unavailable, payment is required to use this feature"
   ErrorMessage {title, message} → maybe "" (_ <> ": ") title <> message
+  MultipleErrors errs → "Multiple errors occured: " <> intercalate ", " (map printQError errs)
```

Comment:
It's a list operation in Haskell, rather than monoidal, which is why I assumed it's not the one to use in PS.